docs/modules/deployment/pages/time-series-storage/timeseries/cortex.adoc

```diff
@@ -0,0 +1,83 @@
+
+= Cortex Time Series Plugin
+
+The Time Series Storage Plugin for Cortex enables {page-component-title} to persist aggregated flow data to Cortex.
```

Comment:
Please change "aggregated flow data" to "time-series performance and latency data".

Reply:
```suggestion
The Time Series Storage Plugin for Cortex enables {page-component-title} to persist time-series performance and latency data to Cortex.
```